extract/community/dcinside_crawler.py

```
@@ -0,0 +1,279 @@
+            comment_elements = soup.select("p.usertxt.ub-word")
+            if not comment_elements:
+                return None
+            comments = '\n'.join(
+                el.text[:-len("- dc App")].strip() if el.text.endswith("- dc App") else el.text
+                for el in comment_elements
+            )
+            return comments
+        
+        # 추천/비추천 크롤링
+        def _get_post_up_down(soup):
+            try: up = soup.select_one("div.up_num_box p.up_num").text
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)